client/components/domains/transfer-domain-step/transfer-restriction-message.jsx

```diff
@@ -87,7 +89,9 @@ class TransferRestrictionMessage extends React.PureComponent {
 				{
 					args: {
 						domain,
-						daysAgoRegistered: moment().diff( moment( creationDate ), 'days' ),
+						daysAgoRegistered: this.props
+							.moment()
+							.diff( this.props.moment( creationDate ), 'days' ),
```

Comment:
Even in a diff call? Ah, magic APIs :)

Anyway, thanks for the tip. Done!

Reply:
Yes, that's consistent across moment.js APIs: whenever a moment.js function accepts a moment as a parameter, it can be in any format: `Moment` instance, `Date` instance, date string, number (unix timestamp), ... That works for `moment().isAfter( m )`, `moment().isBetween( m, n )`, ...